package.json

```diff
@@ -16,7 +16,10 @@
     "prepare": "husky install",
     "codegen:moderation": "graphql-codegen --require dotenv/config --config codegen-moderation.ts",
     "codegen:squid": "graphql-codegen --require dotenv/config --config codegen-squid.ts",
-    "codegen:notifications": "graphql-codegen --require dotenv/config --config codegen-notifications.ts"
+    "codegen:notifications": "graphql-codegen --require dotenv/config --config codegen-notifications.ts",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build",
+    "start-storybook": "node ./.storybook/index.js"
   },
   "dependencies": {
```

Comment:
Is it has impact on bundle size?

Reply:
no, storybook has its own out folder which is unrelated to grill